Library/Homebrew/utils/curl.rb

```diff
@@ -304,7 +304,13 @@ def curl_http_content_headers_and_checksum(
     )
       file = Tempfile.new.tap(&:close)
 
-      specs = specs.flat_map { |option, argument| ["--#{option.to_s.tr("_", "-")}", argument] }
+      specs = specs.flat_map do |option, argument|
```

Comment:
I don't really understand what the code below is doing or why it's needed. A comment might be worth adding?

Reply:
I think it's trying to convert specs to options, but made the strong assumption they are all key value. When I've added a `specs[:insecure] = true` elsewhere it broke this.

Reply:
Ah, gotcha. Maybe worth adding a comment explaining this?